addons/mail/static/src/models/follower/follower.js

```diff
@@ -78,6 +78,7 @@ function factory(dependencies) {
                 method: 'message_unsubscribe',
                 args
             }));
+            this.followedThread.getSuggestedPartners();
```

Comment:
suggested partners should have computed method to adjust from followers

Reply:
So basicly you mean: 

```javascript
fetchUpdateSuggestPartners() {
    return this.followedThread.getSuggestedPartners();
}
```
inside follower.js ?

Reply:
No, I mean the code is made with an imperative mindset (_"when follower is deleted, it should adjust suggested partners of thread"_), and this doesn't scale well.

I suggest turning suggested partners field into a computed field so that it automatically adjusts suggested partners when something that rely on it has changed, like on the related followers (_"when something has changed on followers of thread, suggested partners must be adjusted"_).